source/plugin/blocks/accessing.rst

```diff
@@ -34,56 +34,128 @@ referenced block is any kind of banner by checking the blocks type.
     import org.spongepowered.api.block.BlockType;
     import org.spongepowered.api.block.BlockTypes;
 
-    public boolean isBanner(Location block) {
-        BlockType type = block.getBlock().getType();
-        return type == BlockTypes.STANDING_BANNER
-                || type == BlockTypes.WALL_BANNER;
+    public boolean isBanner(Location<World> blockLoc) {
+        BlockType type = blockLoc.getBlock().getType();
+        return type.equals(BlockTypes.STANDING_BANNER)
+                || type.equals(BlockTypes.WALL_BANNER);
     }
 
 .. tip ::
+    
+    The function ``==`` could be used in place of ``equals()`` as there is only one ``BlockType`` instance
+    for every block, however it is recommended to use the ``equals()``.
 
-    For every possible type of block, there is only one ``BlockType`` instance. Therefore, simple comparison with ``==``
-    can be used instead of using the ``equals()`` function.
+Block Data Manipulators
+~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``Location`` class also provides means to check for environmental influences on the block (for example lighting and
-redstone power). The following example checks if a block is a regular chest directly or indirectly powered with redstone.
+The data of a block is held as a ``DataManipulator``, similar to other parts of the API. This is the container that holds information on components of our block such as the orientation of a block, specific types (stone vs. granite), and so on. Checking the values of these manipulators is easy, you just need to check the block's direction ``DirectionalData``.
```

Comment:
This needs a little work: "Checking the values of these manipulators is easy, you just need to check the block's direction ``DirectionalData``."

Maybe "Checking the values of these manipulators is simple; for example, if you want to check the block's direction, query for its ``DirectionalData``."